schema/types/volume.go

```diff
@@ -138,6 +164,7 @@ func VolumeFromString(vp string) (*Volume, error) {
 		return nil, err
 	}
 	for key, val := range v {
+		val := val
```

Comment:
it's idiomatic actually :/
https://golang.org/doc/effective_go.html
search for "req := req"